src/code/components/chromosome.js

```diff
@@ -2,21 +2,41 @@ import React, {PropTypes} from 'react';
-import AlleleView from './allele';
 import GeneticsUtils from '../utilities/genetics-utils';
 import {getChromosomeDescriptor} from '../fv-components/fv-chromosome-image';
+
 /**
- * View of a single chromosome, with optional labels, pulldowns, and embedded alleles.
+ * View of a single chromosome, with optional labels and pulldowns.
  *
- * Defined EITHER using a Biologica Chromosome object, OR with a Biologica organism,
- * chromosome name and side.
+ * @param {Object} chromosome - the chromosome to be represented. If null, the chromosome will be empty
+ * @param {Object} chromosomeDescriptor - if no chromosome is provided, a descriptor can be passed to create an empty, unlabeled chromosome
+ * @param {Object} ChromosomeImageClass - the class to use for representing the actual chromosome, as opposed to its labels
+ * @param {string[]} userChangeableGenes - the genes that can be changed using a drop down menu by the user
+ * @param {string[]} visibleGenes - genes which can be viewed by the user, but not changed
+ * @param {string[]} hiddenAlleles - individual alleles within a gene which cannot be viewed by the user
+ * @param {boolean} small - determines whether a gamete chromosome is represented, or a full-sized chromosome in a genome
+ * @param {boolean} editable - true if genes in the chromosome can be changed by a drop-down
+ * @param {boolean} selected - true if the chromosome has been selected by the user
+ * @param {function} onAlleleChange - this callback is invoked if the user changes an allele using a dropdown 
+ * @param {function} onChromosomeSelected - this callback is invoked if the user selects this chromosome
+ * @param {boolean} showLabels - false if the labels on a non-empty chromosome should be hidden
+ * @param {boolean} labelsOnRight - true if the labels should be on the right of the chromosome; otherwise, they are placed on the left
+ * @param {string} orgName - the unique name of the organism which the chromosome is a part of (e.g. "targetMother")
+ * @param {number} height - the height in pixels of the rendered chromosome
+ * @param {object} displayStyle - this style will be applied to the actual chromosome, as opposed to its labels
  */
-
-const ChromosomeView = ({chromosome, ChromosomeImageClass=ChromosomeImageView, userChangeableGenes = [], visibleGenes = [], hiddenAlleles = [], small = false, editable = true, selected = false, onAlleleChange, onChromosomeSelected, showLabels = true, showAlleles = false, labelsOnRight = true, orgName, isSelectedEmpty, height, displayStyle = {}}) => {
+const ChromosomeView = ({chromosome, chromosomeDescriptor, ChromosomeImageClass=ChromosomeImageView, userChangeableGenes = [], visibleGenes = [], hiddenAlleles = [], small = false, editable = true, selected = false, onAlleleChange, onChromosomeSelected, showLabels = true, labelsOnRight = true, orgName, height, displayStyle = {}}) => {
+  chromosomeDescriptor = chromosomeDescriptor || getChromosomeDescriptor(chromosome);
```

Comment:
Adding a `chromosomeDescriptor` here is necessary since empty chromosomes are now passed as null. The descriptor lets the `ChromosomeImage` render the correct empty chromosome